src/api/app/controllers/webui/groups/bs_requests_controller.rb

```diff
@@ -28,6 +43,33 @@ def set_group
       def request_method
         REQUEST_METHODS[params[:dataTableId]]
       end
+
+      def set_bs_request
+        return unless Flipper.enabled?(:request_index, User.session)
+
+        @bs_requests = @user_or_group.requests
```

Comment:
@danidoni its confusing that the variable is called `@user_or_group`, its always a group as far as I can see. I would rename it everywhere in the controller.

Reply:
Yeah, you are right. I'll rename this

Reply:
ah okay, alright then lets follow up once this is rolled out

Reply:
Tracked in this card https://trello.com/c/c5dsp0PV/109-rename-userorgroup-variable-in-groups-bsrequestscontroller

Reply:
` @user_or_group` is set in this controller and only used in this controller. It's also *never* a user. Please change this.

Reply:
did it myself...